cli/src/action/circuit/mod.rs

```diff
@@ -119,3 +119,75 @@ fn vote_on_circuit_proposal(
 ) -> Result<(), CliError> {
     unimplemented!()
 }
+
+pub struct CircuitListAction;
+
+impl Action for CircuitListAction {
+    fn run<'a>(&mut self, arg_matches: Option<&ArgMatches<'a>>) -> Result<(), CliError> {
+        let args = arg_matches.ok_or_else(|| CliError::RequiresArgs)?;
+
+        let url = args.value_of("url").unwrap_or("http://127.0.0.1:8080");
+
+        let filter = args.value_of("member");
+
+        list_circuits(url, filter)
+    }
+}
+
+fn list_circuits(url: &str, filter: Option<&str>) -> Result<(), CliError> {
+    let client = api::SplinterRestClient::new(url);
+
+    let circuits = client.list_circuits(filter)?;
+    println!(
+        "{0: <80} | {1: <30}",
+        "CIRCUIT ID", "CIRCUIT MANAGEMENT TYPE",
+    );
+    circuits.data.iter().for_each(|circuit| {
+        println!(
+            "{0: <80} | {1: <30}",
+            circuit.id, circuit.circuit_management_type,
+        );
+    });
```

Comment:
Can we put a separator between the headers (CIRCUIT_ID and CIRCUIT MANAGEMENT TYPE) and the data itself?